src/turnkeyml/tools/management_tools.py

```diff
@@ -216,23 +234,28 @@ def run(
 
 class ModelsLocation(ManagementTool):
     """
-    Prints the location of the turnkeyml built in models corpus.
+    Prints the location of the turnkeyml built in models corpora.
+
+    This is especially useful for when turnkey was installed from PyPI
+    with `pip install turnkeyml`. Example usage in this context:
+        models=$(turnkey models-location --quiet)
```

Comment:
Outside the scope, but I really with there was an easier way to do this.
(e.g. merging model directories during installation)

Reply:
+1!